runtime-modules/content/src/lib.rs

```diff
@@ -456,6 +476,23 @@ pub struct Person<MemberId> {
     controlled_by: PersonController<MemberId>,
 }
 
+/// Payment claim by a channel
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Default, Copy, Clone, PartialEq, Eq, Debug)]
+pub struct PullPaymentElementRecord<ChannelId, Balance, HashType> {
```

Comment:
`Record` is a postfix only used for types that are used as storage values, this is not.